File: Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Indexes/StopIndex.cs
using Raven.Client.Documents;
using Raven.Client.Documents.Operations.Indexes;

namespace Raven.Documentation.Samples.ClientApi.Operations.Indexes
{
	public class StopIndex
	{
		private interface IFoo
		{
            /*
            #region stop_1
            public StopIndexOperation(string indexName)
            #endregion
            */
        }

        public StopIndex()
		{
			using (var store = new DocumentStore())
			{
                #region stop_2
                store.Maintenance.Send(new StopIndexOperation("Orders/Totals"));
                #endregion
			}
		}
	}
}

File: Documentation/4.0/Samples/java/src/test/java/net/ravendb/ClientApi/Operations/Indexes/StopIndex.java
package net.ravendb.ClientApi.Operations.Indexes;

import net.ravendb.client.documents.DocumentStore;
import net.ravendb.client.documents.IDocumentStore;
import net.ravendb.client.documents.operations.indexes.StopIndexOperation;

public class StopIndex {

    private interface IFoo {
        /*
        //region stop_1
        public StopIndexOperation(String indexName)
        //endregion
        */
    }

    public StopIndex() {
        try (IDocumentStore store = new DocumentStore()) {
            //region stop_2
            store.maintenance().send(new StopIndexOperation("Orders/Totals"));
            //endregion
        }
    }
}
